pkgs/by-name/mo/monkeysAudio/package.nix

```diff
@@ -5,15 +5,20 @@
 }:
 
 stdenv.mkDerivation (finalAttrs: {
-  version = "10.76";
+  version = "10.81";
   pname = "monkeys-audio";
 
   src = fetchzip {
     url = "https://monkeysaudio.com/files/MAC_${
       builtins.concatStringsSep "" (lib.strings.splitString "." finalAttrs.version)}_SDK.zip";
-    hash = "sha256-ropQZraOombq6zG5vXU/kBtQggy30ErbU79gbEtrIxs=";
+    hash = "sha256-sI2u+H/ewva9r+g5xSNqal0DMul+a9Y4FV6dEzulvSI=";
     stripRoot = false;
   };
+
+  env.NIX_CFLAGS_COMPILE = toString [
+    "-std=c++11"
+  ];
```

Comment:
Would be nice to comment here what kind of errors have your experienced that made you put this, and how did you realize this is the way to fix the issue you experienced.